internal/buildscripts/packaging/tests/package_test.py

```diff
@@ -62,6 +64,23 @@ def get_libcap_command(container):
     else:
         return "zypper install -y libcap-progs"
 
+@pytest.mark.parametrize(
+    "distro",
+    [pytest.param(distro, marks=pytest.mark.tar) for distro in TAR_DISTROS]
+)
+def test_tar_collector_package_install(distro):
+    pkg_path = get_package(distro, PKG_NAME, PKG_DIR)
+    assert pkg_path, f"{PKG_NAME} package not found in {PKG_DIR}"
+    pkg_base = os.path.basename(pkg_path)
+
+    with run_distro_container(distro) as container:
+
+        copy_file_into_container(container, pkg_path, f"/test/{pkg_base}")
+        run_container_cmd(container, f"tar xzf /test/{pkg_base} -C /tmp")
+        run_container_cmd(container, f"test -d /tmp/bin")
+        run_container_cmd(container, f"test -d /tmp/agent-bundle")
+        run_container_cmd(container, f"test -f /tmp/bin/otelcol")
+        run_container_cmd(container, f"test -f /tmp/bin/translatesfx")
```

Comment:
Do these paths need to be updated for the new `splunk-otel-collector` base dir?

Reply:
yes they do. Fixing...